docs/source/models/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -229,17 +229,43 @@ SQ-LNS effects on wasting will persist until 18 months of age and effects on stu
 Targeted SQ-LNS
 ~~~~~~~~~~~~~~~
 
+There is not a gold standard currently for how to target SQ-LNS. This 
```

Comment:
Let's cite and maybe summarize the current WHO guidelines on targeting SQLNS! There is definitely a recommended practice, it's just a little vague!